README.md

```diff
@@ -0,0 +1,40 @@
+# Hollowverse Infrastructure
+
+[Terraform](https://terraform.io/) configuration files for Hollowverse infrastructure.
+
+For an overview of the infrastructure, please refer to the [Infrastructure Overview](https://github.com/hollowverse/hollowverse/wiki/Infrastructure-Overview) page on the wiki.
+
+## Making Changes to the Infrastructure
+
+Before making any changes to the infrastructure of Hollowverse, we strongly recommend that you take a look at the code in [`hollowverse/infrastructure`](https://github.com/hollowverse/infrastructure) and read it carefully to have a basic idea on what resources are used and how they are connected to each other. The syntax of Terraform configuration files is pretty simple and readable.
+
+You should also refer to [the Terraform configuration documentation](https://www.terraform.io/docs/configuration/index.html) for a more comprehensive understanding of how to write these files.
+
+### Manual Update Workflow
+
+To update the Hollowverse infrastructure, you need to have administrator privileges on the Hollowverse AWS account. You will also need to install Terraform. We currently use the [version 0.11.7](https://releases.hashicorp.com/terraform/0.11.7/) and we recommend that you use the same version.
+
+Additionally, you need to have access to any sensitive resources that are required to initialize Terraform on your computer. These include the Terraform state that is used in production, which is stored in a private S3 bucket, as well as any other passwords or SSH keys used in the Terraform code.
```

Comment:
>you need to have access

Does this mean I have to know the passwords or that I should have the right authorizations?

Reply:
You will need to know the password. It will be used to compare the remote state with the local changes. If the password used locally does not match the once in the remote state, the database user will be updated to use the new password.

Reply:
Regardless, the user executing the plan will already have access to the remote state, which will contain the passwords in plain text. So there is no additional security risk.